benchmark.sh

```diff
@@ -1,12 +1,8 @@
 #!/bin/bash
 set -xeuo pipefail
-echo running benchmark harness...
-if [ $# -ge 1 ]; then
-  echo with repo auth
-  TMP=`mktemp -d`
-  hhvm --hphp -t hhbc --module=generated --module=lib --module=test --output-dir $TMP
-  hhvm -d hhvm.repo.authoritative=true -d hhvm.repo.central.path=$TMP/hhvm.hhbc test/test.php bench
-else
-  echo without repo auth
```

Comment:
Discussing with Fred, HHVM without repo auth is not well optimized so running the benchmark without it is likely to lead to poor optimizations.